Tensile/KernelWriter.py

```diff
@@ -1307,7 +1307,9 @@ def setupNewTile(self, kernel, tensorParametersA, tensorParametersB, isPap, isOp
       #    but since there are some vgpr value is used in the later lwaFirstOffset (when not OptNLL, such as "lwoT")
       #    so we still do this part when "isPap & not OptNLL"
       # 2. if tile edge, then we still need to add all these codes even isPap
-      self.dontAppendCode = isPap and isOptNLL and (not needShift)
+
+      # self.dontAppendCode = isPap and isOptNLL and (not needShift)
```

Comment:
Do we need to keep original code here?
Don't we need to update above comment regarding OptNLL since OptNLL is no longer used here?